src/pages/library/datasetBuilder/CohortEditor.test.ts

```diff
@@ -187,7 +185,7 @@ describe('CohortEditor', () => {
     const onStateChange = jest.fn();
     datasetBuilderCohorts.set([]);
 
-    render(h(CohortEditor, { onStateChange, datasetDetails, originalCohort }));
+    render(h(CohortEditor, { onStateChangeHandler: onStateChange, datasetDetails, originalCohort }));
```

Comment:
nit:  Handler postfix is good for type name, but not needed for "event prop" name.  onEventX is sufficient.

Reply:
OK, I usually match up type names with variable names, so I renamed the props based on that convention. I can revert the renaming for this `onX` property.